src/main/resources/db/migration/V9__Create_email_subscription_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE email_subscription
+(
+    id                 bigserial                not null,
+    email              text                     not null,
+    unsubscribed       boolean                  not null default true,
```

Comment:
Why is this true by default? Is it because the existence of a record implies that this email is unsubscribed? (I think I answered my own question after asking)

Reply:
Correct

Reply:
Defaults are not really needed because we explicitly set the value in the code. However, this makes it easier if we ever use other means to populate the table.